packages/client/api/README.md

```diff
@@ -2,6 +2,11 @@
 
 This package is a part of FluenceJS, the official implementation of the Fluence Peer in typescript. See the [FluenceJS repo](https://github.com/fluencelabs/fluence-js) for more info
 
+## Notes
+
+This package just reexports everything from main library for backward compatibility
+`export * from '@fluencelabs/js-client';`
+
```

Comment:
is this all needed? I rememeber we agreed ot just deprecate this all

Reply:
We will remove this package after merge